keps/sig-node/20180906-quotas-for-ephemeral-storage.md

```diff
@@ -613,19 +627,111 @@ appropriate end to end tests.
 The following criteria applies to
 `LocalStorageCapacityIsolationFSMonitoring`:
 
-Alpha
+### Phase 1: Alpha (1.15)
 
 - Support integrated in kubelet
+- Alpha-level documentation
+- Unit test coverage
+- Node e2e test
 
-Beta
+### Phase 2: Beta (target 1.16)
```

Comment:
nit: We need the user feedbacks from alpha feature, I doubt we can get that to promote it to the beta in 1.16. But I am ok to target here.